### PR TITLE
move to new csproj packaging

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: src/NServiceBus.Newtonsoft.Json/NServiceBus.Newtonsoft.Json.csproj [hunk 1]
```diff
@@ -2,23 +2,35 @@
...
     <AssemblyOriginatorKeyFile>$(SolutionDir)NServiceBus.snk</AssemblyOriginatorKeyFile>
-    <DebugType>full</DebugType>
-    <DebugSymbols>True</DebugSymbols>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <OutputPath>..\..\binaries\</OutputPath>
-    <UpdateAssemblyInfo>true</UpdateAssemblyInfo>
     <NoWarn>0419</NoWarn>
-    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <Description>Newtonsoft.Json integration for NServiceBus.</Description>
```
Comment (timbussmann) [inline src/NServiceBus.Newtonsoft.Json/NServiceBus.Newtonsoft.Json.csproj]: shouldn't this element be in the next property group? It seems like it's related to the nuget package?

File: src/Tests/Tests.csproj [hunk 2]
```diff
@@ -10,13 +10,14 @@
...
   <ItemGroup>
     <Compile Remove="NServiceBus.Newtonsoft.Json.approved.cs" />
-    <PackageReference Include="ApiApprover" Version="6.1.0-beta2" />
-    <PackageReference Include="ObjectApproval" Version="1.6.0" />
-    <PackageReference Include="PublicApiGenerator" Version="6.1.0-beta2" />
-    <PackageReference Include="NUnit" Version="3.7.1" />
-    <PackageReference Include="ApprovalTests" Version="3.0.13" />
-    <PackageReference Include="ApprovalUtilities" Version="3.0.13" />
-    <PackageReference Include="Mono.Cecil" Version="0.10.0-beta6" />
+    <Compile Remove="NServiceBus.Newtonsoft.Json.received.cs" />
```
Comment (timbussmann) [inline src/Tests/Tests.csproj]: why are the approved/received files here ending with cs and not txt like the other packages?

**Comment by timbussmann** [comment]
also seems there is a code inspection failing on the build server

File: src/Tests/Tests.csproj (hunk 2 above)
Comment (SimonCropp) [inline src/Tests/Tests.csproj]: cs is the default output of api approver

**Comment by timbussmann** [comment]
addressed most changes in this PR in here as well: https://github.com/Particular/NServiceBus.Newtonsoft.Json/pull/25. Are you ok with closing this PR?

**Pull request closed without merging**